examples/browser-webpack/package.json

```diff
@@ -11,13 +11,17 @@
     "babel-core": "^6.24.1",
     "babel-loader": "^7.0.0",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
     "json-loader": "^0.5.4",
-    "react": "^15.5.4",
     "react-hot-loader": "^1.3.1",
     "webpack": "^2.5.1",
     "webpack-dev-server": "^2.4.5"
   },
   "dependencies": {
+    "ipfs": "^0.26.0",
```

Comment:
We still want to use IPFS from this repo -  it needs to be something like:

```
"dependencies": {
   "ipfs": "../../../src/core",
   ....
}
```

Reply:
This one should be correct
```
"dependencies": {
   "ipfs": "file:../../",
}
```

Reply:
Oh, nice! Much better